src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,61 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React, { useCallback } from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  query: string,
+  selectedTodos: string,
+  onChangeSelectedTodos: (selectedTodos: string) => void,
+  onChangeQuery: (query: string) => void,
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<Props> = ({
+  query,
+  selectedTodos,
+  onChangeSelectedTodos,
+  onChangeQuery,
+}) => {
+  const reset = useCallback(() => onChangeQuery(''), []);
```

Comment:
maybe do it in parrent component?

Reply:
You don't need this function, just reset in `onClick` as you did with setting the query